### DISCUSSION TITLE
Dropdown menu

### DISCUSSION THREAD

**Discussion by Sup3rlativ3** [thuliteio/doks#1214] [General]
Hi,

I was wondering if it's possible to have a drop down menu similar to the site https://carsten-nichte.de/

I have had a look but I can't seem to find a way to make it work.

any help would be appreciated.

Thanks in advance!

**Comment by h-enk** [comment]
In short: use `identifier` and `parent` — see for example: https://github.com/gethyas/getdoks.org/blob/v0.4.2/config/_default/menus.toml#L63-L73

Note to self: to be documented!

**Reply by Sup3rlativ3** [reply]
That has worked, Thank you <3

I get the first level but not the second, is that expected?


Here is my menus.en.toml
```
[[main]]
  name = "Builds"
  url = "/builds"
  weight = 15
  identifier = "builds"

[[main]]
  name = "Healer"
  url = "/builds/healer"
  weight = 20
  parent = "builds"
  identifier = "healer"

[[main]]
  name = "Advanced Healer"
  url = "/builds/healer/advanced-healer"
  weight = 30
  parent = "healer"
  identifier = "advanced-healer"

[[main]]
  name = "Standard Healer"
  url = "/builds/healer/standard-healer"
  weight = 20
  parent = "healer"
  identifier = "standard-healer"
```

Only the Healer menu is shown in the dropdown, not the advanced-healer.

**Reply by Sup3rlativ3** [reply]
Okay, so I did some digging and I'm guessing that because Bootstrap doesn't support nested dropdown doks doesn't either?

**Reply by h-enk** [reply]
No, it's because Doks does not support this — yet. Improved main navigation (including unlimited sub-navigation levels) will be part of the upcoming Doks release. See also the Hyas Roadmap: https://github.com/orgs/gethyas/discussions/462

**Reply by Sup3rlativ3** [reply]
That's awesome to hear, thank you very much!
